go.mod

```diff
@@ -4,15 +4,18 @@ go 1.20
 
 require (
 	github.com/containers/gvisor-tap-vsock v0.5.0
+	github.com/go-chi/chi v1.5.4
```

Comment:
Do we need the older `chi` or can this work if the v5 used by the rest of the code?

Reply:
I think we were already using v5? for some reason this got moved from the indirect dep list to the direct dep by `go build` or `go get`, not sure which.